tests/components/apcupsd/test_config_flow.py

```diff
@@ -0,0 +1,168 @@
+"""Test APCUPSd setup process."""
+from copy import copy
+from unittest.mock import patch
+
+import pytest
+
+from homeassistant import config_entries, data_entry_flow
+from homeassistant.components.apcupsd import DOMAIN
+from homeassistant.const import CONF_HOST, CONF_PORT, CONF_RESOURCES
+from homeassistant.core import HomeAssistant
+
+from . import MOCK_MINIMAL_STATUS, MOCK_STATUS
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture(name="config_entry")
+def config_entry_fixture():
+    """Create hass config_entry fixture."""
+
+    return MockConfigEntry(
+        version=1,
+        domain=DOMAIN,
+        title=MOCK_STATUS["MODEL"],
+        data={CONF_HOST: "test", CONF_PORT: 1234},
+        options={},
+        unique_id=MOCK_STATUS["SERIALNO"],
+        source=config_entries.SOURCE_USER,
+    )
+
+
+async def test_flow_works(hass: HomeAssistant, config_entry: MockConfigEntry):
```

Comment:
Please type the whole signature when adding type annotations.
```suggestion
async def test_flow_works(hass: HomeAssistant, config_entry: MockConfigEntry) -> None:
```

Reply:
Got it, made changes to all test functions :)